.github/workflows/docs-pr.yml

```diff
@@ -3,12 +3,15 @@ name: "Docs / Build PR"
 # see https://sphinx-theme.scylladb.com/stable/deployment/production.html#available-workflows
 
 on:
+  push:
+    paths:
+      - "docs/**"
+      - ".github/workflows/docs-pr.yml"
   pull_request:
-    branches:
-      - master
-      - 'branch-**'
+    types: [opened, synchronize, reopened]
```

Comment:
According to [docs](https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#pull_request), `[opened, synchronize, reopened]` is the default. Is there a reason for specifying it explicitly?

Reply:
removed